### DISCUSSION TITLE
Windows 10 install issue |  can't open any image

### DISCUSSION THREAD

**Discussion by Kai01W** [ImageMagick/ImageMagick#4674] [Help]
Hi, 

I installed the latest binary on Windows 10 but when I try to open any image or even run the test (as per the download page):
`magick logo: logo.gif`
I get this: 
```
magick: unable to open image 'logo:': Invalid argument @ error/blob.c/OpenBlob/3527.
magick: no decode delegate for this image format `' @ error/constitute.c/ReadImage/572.
```
I downloaded the **_Visual C++ Redistributable Package_** as recommended but that did not change anything.
This is my -version output:

```
C:\Program Files\ImageMagick-7.1.0-Q16-HDRI>magick -version
Version: ImageMagick 7.1.0-19 Q16-HDRI x64 2021-12-22 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib
Compiler: Visual Studio 2019 (192930138)

```
Any ideas?

 thanks

**Comment by urban-warrior** [comment]
Unfortunately, we cannot reproduce the problem.  We did a fresh install under Windows 10 and your command worked fine.  Try reinstalling with the latest release download and see if the problem persists.

**Reply by BigMouch** [reply]
I have the same error after compiling and running the source code. The main reason is that the file path of the module cannot be found. windows cannot find the module folder if it is not installed by the installation package. No concrete solution has been found. The suspicion is that some environment variable was set during installation.

**Comment by hydGuy** [comment]
tried installing imageMagick , convert.exe is not present in the image magick folder and none of the commands are working, i tried installing the following binaries from admin account
ImageMagick-7.1.1-29-Q16-HDRI-x64-dll.exe
ImageMagick-7.1.1-29-Q16-x64-dll.exe
ImageMagick-7.1.1-29-Q16-x64-static.exe

**Reply by dlemstra** [reply]
With ImageMagick 7 you want to use `magick` intead of `convert`.

**Reply by hydGuy** [reply]
got it , magick was pesent and it works, thank you very much, now my only problem is telugu unicode characters dont display accurately irrespective of the font I use, i shall put it up in the right section
